### ISSUE TITLE
`object-literal-shorthand` disables `object-literal-sort-keys`

### ISSUE THREAD

**Issue by despairblue** [palantir/tslint#2205]
### Bug Report

- __TSLint version__: `4.4.2`
- __TypeScript version__: `2.1.5`
- __Running TSLint via__: CLI (although I also tested it with atom)

#### TypeScript code being linted

```ts
const foo = {
  b: 2,
  a: 1,
}

const a = 1
const bar = {
  b: 2,
  a,
}

export default () => foo === bar
```

with `tslint.json` configuration:

```json
{
  "extends": "tslint:recommended"
}
```

#### Actual behavior

```
src/test.ts[3, 3]: The key 'a' is not sorted alphabetically
```

#### Expected behavior

```
src/test.ts[3, 3]: The key 'a' is not sorted alphabetically
src/test.ts[9, 3]: The key 'a' is not sorted alphabetically
```
